libs/sdk-ui-dashboard/src/presentation/dashboardContexts/types.ts

```diff
@@ -89,6 +89,7 @@ export type InsightMenuItemsProvider = (
     widget: IInsightWidget,
     defaultItems: IInsightMenuItem[],
     closeMenu: () => void,
+    renderMode: "edit" | "view",
```

Comment:
Please use the `RenderMode` type for this in case another mode gets added later.